engine/apps/schedules/models/custom_on_call_shift.py

```diff
@@ -250,12 +281,16 @@ def convert_to_ical(self, time_zone="UTC"):
             event_ical = None
             users_queue = self.get_rolling_users()
             for counter, users in enumerate(users_queue, start=1):
+                rotation_ical = ""
                 start = self.get_next_start_date(event_ical)
                 if not start:  # means that rotation ends before next event starts
                     break
                 for user_counter, user in enumerate(users, start=1):
                     event_ical = self.generate_ical(user, start, user_counter, counter, time_zone)
-                    result += event_ical
+                    rotation_ical += event_ical
+                # if event has already been started, add rotation ical to final ical result
+                if start >= self.rotation_start:
```

Comment:
Couldn't we add the event anyway? (even if there is no instances occurring for a while) Wondering how we would eventually add it to the calendar otherwise (ie. how we would trigger the addition when needed)

Reply:
Anyway I reverted this changes because I think it's better to do updates for ical generation in another PR.